### PR TITLE
Change FilterBox With URL 

### PR DESCRIPTION
**Pull request by frimpongopoku**
Filterbox now changes version with  **URL**
The default version of the horizontal filter box is the one that doesnt use bubbles. So, our everyday-routes will use the bar with no-bubbles.
To active the bubble version in any community, use the following URLS

`wayland/actions-demo/filter/bubble`
`wayland/events-demo/filter/bubble`
`wayland/stories-demo/filter/bubble`
`wayland/services-demo/filter/bubble`

**NB: It works for all communities, "wayland" could be any community**

### CODE REVIEW

**Comment by BradHN1** [comment]
@frimpongopoku - the way we discussed doing this was with a parameter, not the url.  
community.massenergize.org/wayland?option=1  would set option 1 (bubbles or whatever), throughtout the site and session.  Can you do it that way?

**Comment by frimpongopoku** [comment]
Hi @BradHN1 , I tried to implement exactly what was discussed, but found out it wasnt possible due to a few reason. 
1. First, the question mark in the route you want, are  only for `**GET**` form submissions on a backend served pages, it doesnt work in a front end framework using react-router like we use. 
2.  Secondly,  we cannot have such a route like you described in your comment. Mainly because all the routes are predefined from the backend as you already know, and the slashes that look cool and formal are already taken. 
Take an example: 

`wayland/actions/filter?option=1` or anything of such sort
That wont work because there is a route that is already defined to expect `wayland/actions/<a number>/` which takes a user to the individual action page. So putting `/filter?=something`  react takes that whole word as a number and it throws a very nasty error 😄 

So I merely had to options, restructure the whole routing already being used (which just didnt make sense) 
Or just create unique routes from what I am given. Thats why I made things that way.

In short, how we wanted to use the question mark in the URL *wont work*.  We can only work with slashes. However, I can easily change the words `demo, filter, and bubble`  in the structure `wayland/services-demo/filter/bubble`  to whatever you recommend 🤗

**Comment by BradHN1** [comment]
Hello @frimpongopoku  - sorry for slow response but I'm on vacation this week.
I get what you say, that we can't do it with a get parameter, which is too bad.

I would like you ask you to change it such that by accessing one URL once: (/_community_/OptionA) that the first option is used for the entire session (store it as a state variable in community data redux), and the second option is enabled by visiting the url: /_community_/OptionB

We dpn't want the words filter/bubble in the url that is entered, and we want the tester to only have to enter the special url once at the beginning.  Can you do that?

**Comment by frimpongopoku** [comment]
Hi @BradHN1, yeah, the session idea is easily possible. However, I wld need to  check the availability of the `community/optionB` route though... Will work on it...  🤗
